DataFormats/L1Trigger/src/classes_def.xml

```diff
@@ -280,8 +280,8 @@
 
   <class name="edm::Ptr<l1t::L1Candidate>"/>
 
-  <class name="l1t::Vertex" ClassVersion="3">
-    <version ClassVersion="3" checksum="514139083"/>
```

Comment:
Any reason not to keep the class version history available and just add the new checksum (v4) in addition to v3? I don't know the CMSSW rules surrounding this type of thing, but I often see the history of class versions still around in `classes_def.xml` files.

Reply:
@cecilecaillol   as @aperloff suggested please keep the class version history.  Please also see e.g. https://github.com/cms-sw/cmssw/pull/20651#discussion_r142871184